deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -3,21 +3,42 @@ kind: ClusterServiceVersion
+  namespace: placeholder
 spec:
+  version: 0.0.1
+  maturity: alpha
+  provider:
+    name: Red Hat, Inc.
+  maintainers:
+  - name: Red Hat, Inc.
+    email: customerservice@redhat.com
+  icon:
+  - base64data: iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAAH30lEQVRYR51Zb1ATZxp/l3+2JOBdGkfUD3W8aMGOboWe5fJnGhDSQOlcbrw5erQdIcDxgQklLDSi1jtPhKywYXKDN52KQWqgokwd73oGKRhAEqsNelEoeueHOnNDTVmY82BNsU1y86a7mTUm2Q3vt9339/ye37553ud93icIWOUoKipKLy4urti+ffs7YrFYAgB4DgCQQtM9AQB8T5Lk/ZmZGevAwECPw+FYWo0rJB4jpVKZpNPpmiUSSRMAIC0eWwRB/udyuYxarfY4AMDH15avwCS73f6pSCT6LV/iWLiFhYVP8/Pz3+MjlFOgXq9Xl5eX/wMAkBDm9AlJkmP9/f39FEVdOXv27EPWfEJpaemLtbW1e5OSkvYJBIJtAIBwX77e3t43TCbTaKyPiSnw8uXLH2dkZFSzCfx+v6ejo0PT19d3HQAQ4LmiSTiOlxYWFpoTExNfYNt4PJ6/qFSq96PxRBXocDiuCoVCOcvwSUtLi+r8+fPjPEVFhLW3t/9OpVL1AwASGcDy8vIXMplMFckgokCHw3FNKBTmMgYURf1TKpW+yidmeIpPvnPnzl2/37+FJfKKTCbbE27/jECHw3FWKBSWsgz/JpPJfs3TcVywycnJvrS0tDLGaH5+vrugoOCpkHpKYH19fX5FRUUoaJeWlobkcnlRDK8w78HNw8Qiw8c8/wAA8MdS7XA4PhcKhW8ymHPnzv3y2LFjLuaZLTDR7XZ/DwBIgpMJCQn3d+zYsTUauUgkSj948MNHHEtGYZheyLWs09PT93w+H9zpcKygKJrKfFhIoN1u/0wkEv2GBvlRFH0eAABPhIijpKQkOy9vzxSH82UM03Mm9M2bNz938eJFikllU1NTH2u12hrIHRSYk5OTbLFYQmKGhobeNRgMfbGcV1VVNWVlvQxPhahjfn7+iNHY+ieuFYTzOI6/o1arrQwWRVG4y/1BgYODg8TWrVsb6MnHKIoKuEjb2nBbSkqKOhbu+HGj0OPxwJXhNegQWwPBU1NTTVqttiMokD1x6dKlt5ubmwe4GAnC9A0AyIvRcchDDKvfwMXDnm9ra/t9cXExzJEAnt07d+5ciyiVygyz2fwtDQygKAp35o9cxATRCXdn1EQ/OztT1d3dfYqLJ2w+xe12rzDvmpqahMjVq1cPpqent8CXFEV9LZVKX+ZDShCdMY85DNPDbMC7amF8Op3O+wKB4Bfw2eVyVSGjo6OzYrE4E76YmJio1el0f+USuGXLlrW1tbr/cuEizAfu3r333smTH0XdgGazuVGpVLZDW5IkRxG32w0LyWCuam9vz7Rarfe4HGdmZv6qurrGyYWLNP/ddw//jOP4H6PZlpWV5RgMBiZRL0KBML0kQwMUReEOipr7GNKGhsaGTZs2EasQ+JhO3LHCgx2HXigwFOz0BoHHU8zR2Gi4vGFDRsTqI5bh4OC57GvXrt3ioE+mFw3CfFAgDORgMcpXIEF0fgMAiJFinpVAUcvjhw9/qOT6eHivYe1kPxQIz99gcuT7E2s0ezUKhRxHkJ/STCDwU7EAnwMBAC9Qz6QfDNPzCh+ohdYEKVegQLgb18Kn1tbWlwYGBv7F4yujQZ4niM7H4ZPT09PVPT2nuvnwlpaWZh84cIA54x8hIyMjX61btw4Wo+DGjRva6urqHj5EkTAlJZo9eXmvj4TNeTBMn8GXs6urq1mhULRCPEmS15Gurq56hULRCV94vd6bubm5OXzJwnFHjx67kJqaqmG/t1o/2XTr1q05vpy3b9+eCwQCwSNyfHxch8jl8p+fOHFikSaAZRZMOTGLzGjOCKITHlPM5R14vSs9hw7t1/IVB9MdawcDDMN+xhQLsOKARSKw2Wya/fv3X4yDNAjduHFjKoY1sSuXJximh90Gvjc/YDQaPygqKsJp38soiqYFBVosltacnJxmemIJRdH0eAXW1NQUbtuWOczYjYwMK202W1w3QDq9BH8Bl8t1uLKy8mhQIGxpmM3mUIIeGhraazAYPotHZGsrPrRmTcob0CYQCLgbGxteicf+9OnTdbt27TIzNiiKBosNdsl/ntXa+JEu+TnLLoaQIDqZIzNAEO3Cubm5Z9JNNMEwPGw2Wyg8FhYWevPz88shPuqliaKoaalUuoPnKiB0fQhmZu58YLFYgtUI3+F2u9kn0w8oisLYDW7UpzK+Xq9/vby8fIwhXlpa+kQul+/j4+jIkZYrXi+FGI1teXzwDMbpdNoFAkHoCLRYLLlmsxm2VYKD8+K+srJycvfu3X+Ixylf7OTkpC0tLS10r/F4PKdUKlUV2z5iye50Or8UCASvMUCKoq5LpVLpavNjuGCJRLLmwoULX4e1PuwymSw/HBtX82h8fPyturq6UCrhu1JsHI7j+9RqtYXdzqMoakQqlRZG4ovZfhseHu5ev359JdvQ5/MtmkymvVardSKOFU3GcfxdtVoNrxNwA4TGxMSEWafT1Uf7WM4GZl1dXXFlZeXfIzQwfSRJ3njw4MFpu93+xZkzZ/7DcgIbmJkKhUKenZ1dKxAIsiIIgA3MQpPJZI/1S3AKpI2T7Xb7AKs1sppfN2SzuLjYl5eXB7MD562Pr8AgOd1EPySRSDDmohWH0kc3b95sq6io6OAjLGqa4euwoKBgrUajqczKyioTi8WwERn+N4SXJMl/z87OnhkcHOwdGxtb5svNxv0f6M31w9Aq28gAAAAASUVORK5CYII=
```

Comment:
what image are we using here?

Reply:
the default image which will be shown if there are no icons
the grey lightning bolt

Reply:
Let's use the default image till we get an icon for the pipelines. We can' tuse the Tekton logo here.